nix/hydra/demo.nix

```diff
@@ -1,177 +1,174 @@
 # A demo using process-compose
+{ inputs, self, ... }:
+{
 
-{ system ? builtins.currentSystem
-, pkgs
-, inputs
-, cardano-cli
-, cardano-node
-, hydra-node
-, hydra-tui
-, self
-, process-compose
-}: {
-  package = process-compose;
-  settings = {
-    log_location = "devnet/logs/process-compose.log";
-    log_level = "debug";
-    environment = {
-      CARDANO_NODE_SOCKET_PATH = "devnet/node.socket";
-      CARDANO_NODE_NETWORK_ID = "42";
-    };
+  perSystem = { pkgs, self', ... }:
```

Comment:
What is the difference between `self` and `self'`?

Reply:
`self'` implicitly takes care of the `system` variable so you can access `self'.packages.x` directly.